# operation: 
ActionType action_type
moveit_msgs/BoundingVolume start
moveit_msgs/BoundingVolume end
---
# Define the result
moveit_msgs/BoundingVolume[] quadrants
geometry_msgs/Pose[] quadrant_poses 
ProcessStatus process_status 
---
# Define a feedback message
ProcessStatus process_status
float32 percent_complete


